bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/AbstractTreeViewer.java

```diff
@@ -83,27 +84,52 @@ public abstract class AbstractTreeViewer extends ColumnViewer {
 	 *
 	 * @see #expandToLevel(int)
 	 * @see #collapseToLevel(Object, int)
+	 * @see #setAutoExpandOnSingleChildLevels(int)
 	 */
 	public static final int ALL_LEVELS = -1;
 
+	/**
+	 * Constant indicating that no level of the tree should be expanded or collapsed
+	 *
+	 * @see #setAutoExpandOnSingleChildLevels(int)
+	 * @since 3.32
+	 */
+	public static final int NO_EXPAND = 0;
+
 	/**
 	 * List of registered tree listeners (element type:
 	 * <code>TreeListener</code>).
 	 */
 	private ListenerList<ITreeViewerListener> treeListeners = new ListenerList<>();
 
 	/**
-	 * The level to which the tree is automatically expanded each time the
-	 * viewer's input is changed (that is, by <code>setInput</code>). A value
-	 * of 0 means that auto-expand is off.
+	 * The level to which the tree is automatically expanded each time the viewer's
+	 * input is changed (that is, by <code>setInput</code>). A value of {code
+	 * NO_EXPAND } means that auto-expand is off.
 	 *
 	 * @see #setAutoExpandLevel
 	 */
 	private int expandToLevel = 0;
 
 	/**
-	 * Indicates if filters should be checked to determine expandability of
-	 * a tree node.
+	 * How many levels to autoexpand in a path of consecutive single children. {code
+	 * NO_EXPAND} will disable the feature, while <code>ALL_LEVELS</code> will
```

Comment:
This inconsistency is not intended. From [this documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javadoc.html#CHDJGIJB), I infer that `{@code ...}` is slightly better fit for our usecase. I will check through the PR to make sure that I consistently use the correct JavaDoc-Tag - unless otherwise warranted

Reply:
IIRC, I used `<code> ... </code>` because the rest of the file used it a lot. I don't believe that refactoring the entire file for this consistency is a project for this PR - We could however choose to create a PR where we change all occurences of ``<code>..  </code>` to `{@code ...}`. If you agree with me, I will open an issue to put this up for discussion

Reply:
No, I did not want to propose to cleanup the entire file. I just wanted to have consistency within the text that this PR touches.